test/Configuration.jl

```diff
@@ -38,4 +40,42 @@ end
     ["--compile=min", "--project=test", "--startup-file=no", "--history-file=no"]
 end
 
+@testset "authentication" begin
+    port = 1238
+    server = Pluto.Configuration.ServerOptions(; port=port, launch_browser=false)
+    options = Pluto.Configuration.Options(; server=server)
+    session = Pluto.ServerSession(; options=options)
+    host = session.options.server.host
+    secret = session.secret
+    @async Pluto.run(session)
+
+    url(suffix) = "http://$host:$port/$suffix"
+    function access_denied(url)
+        try
+            HTTP.get(url)
+            return false
+        catch e
+            return e.status == 403
+        end
+    end
+
+    @test HTTP.get(url("favicon.ico")).status == 200
+
+    routes = [
+        "",
+        "new",
+        "edit",
+        "notebookfile",
+        "notebookexport",
+        "open",
+        "sample",
+        "sample/Interactivity.jl",
+        "statefile",
+        "foo"
+    ]
+    for suffix in routes
+        @test access_denied(url("edit"))
```

Comment:
I think you meant to use `suffix` here :wink:

Reply:
Thanks! Well spotted